doc/examples/ex53/ex53.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#               GMT EXAMPLE 53
+#
+# Purpose:      Illustrate subplot with loops
+# GMT modules:  makecpt, subplot, set, plot, grdimage, clip, coast
+#
+
+gmt begin ex53
+	gmt set PROJ_ELLIPSOID Sphere MAP_ANNOT_OBLIQUE lon_horizontal,lat_parallel,tick_normal FONT_TAG 10p
+	gmt makecpt -Cterra
+	data=$(gmt which -G @Top12Cities.txt)
+	gmt subplot begin 4x3 -Fs6c -A+gwhite+p0.25p+o0.2c+sblack@50 -Baf -R-1500/1500/-1500/1500+uk -X1c -Y0.9c
+		while read lon lat population iso name; do
+			gmt subplot set -A"${name} [${population}M]"
+			gmt grdimage @earth_relief_06m -JA${lon}/${lat}/? -I+m-0.5
+			gmt coast -E${iso}+c
+			gmt grdimage @earth_relief_06m -JA${lon}/${lat}/? -I+d
+			gmt clip -C
+			gmt coast -E${iso}+pfaint -N1/thinner,darkred
+			echo ${lon} ${lat} | gmt plot -Skcity/0.25c -Gred -W0.5p -B0
+		done < $data
+	gmt subplot end
+gmt end show
```

Comment:
The PS file is so large (> 10 Mb).

Reply:
I know.  And that is after I dumbed it down to 6 minutes.  As long as we store PS originals I am not sure what we can do about that - never show anything of high resolution?  Thoughts?

Reply:
Perhaps it's time to use the DVC workflow in the GMT project. The DVC workflow is being used in the PyGMT project. It stores all static images (PNG for PyGMT) on another server, and works pretty well for PyGMT. @meghanrjones Thoughts?

Reply:
BTW, the PNG version of the above was 11 Mb so I made a JPG (3.7M).  Both at 300 dpi default psconvert output.

Reply:
I agree that DVC would be a good solution to the PS size issue @seisman. It would require buy-in from all the core developers since it is one more tool to learn for them and would increase the learning curve a bit for new contributors.